runtime/runtime-params-estimator/src/testbed.rs

```diff
@@ -55,7 +55,7 @@ impl RuntimeTestbed {
             max_total_prepaid_gas: u64::MAX,
             max_number_bytes_method_names: u64::MAX,
 
-            ..Default::default()
+            ..VMLimitConfig::test()
```

Comment:
is feels strange that we're using a 'test' method here  - please comment why

Reply:
Done. Overall intuition is that it is a test structure, it's hard to say something more.

Reply:
sorry - I misread the context.